dumb-init.c

```diff
@@ -68,6 +71,43 @@ void forward_signal(int signum) {
 }
 
 /*
+ * Read /proc and see if there are processes except init(PIDs)
+ */
+signed int process_count() {
```

Comment:
Since this isn't exactly returning the total process count, maybe it'd be better to rename this function to `is_only_process_running`?